horovod/mxnet/handle_manager.cc

```diff
@@ -16,9 +16,9 @@
 #include "handle_manager.h"
 
 namespace horovod {
-namespace MX {
+namespace mxnet {
 
-typedef mxnet::Engine::CallbackOnComplete Callback;
+typedef ::mxnet::Engine::CallbackOnComplete Callback;
 
```

Comment:
Is this necessary given that this is already defined in the header?

Reply:
not necessary. Removed